openapi.yaml

```diff
@@ -16089,6 +16652,99 @@ components:
           x-linode-cli-color:
             None: black
             default_: white
+    LinodeConfigInterface:
+      type: object
+      description: >
+        The Network Interface to apply to this Linode's configuration profile.
+      properties:
+        id:
+          type: integer
+          description: >
+            The ID of the Interface to add to this Linode's configuraiton profile.
+            Use the Interfaces List endpoint to retrieve the Interface ID(s) assigned to this Linode.
+          example: 678
+    LinodeConfigInterfaces:
+      type: object
+      description: >
+        A dictionary of Network Interfaces to add to this Linode's
+        configuration profile.
+
+
+        **Note:** You must create a Network Interface using the Interface Create endpoint prior to updating your configuration profile
+        with a Network Interface. If you assigned Network Interfaces to your Lionde when you initially created
+        it, the Network Interface(s) is automatically added to your Linode's configuration profile and you are not required
+        to add the Network Interface with this field.
+      properties:
+        eth0:
+          $ref: '#/components/schemas/LinodeConfigInterface'
+        eth1:
+          $ref: '#/components/schemas/LinodeConfigInterface'
+        eth2:
+          $ref: '#/components/schemas/LinodeConfigInterface'
+    Interface:
+      type: object
+      description: The Network Interface to apply to this Linode.
+      required:
+      - type
+      properties:
+        id:
+          type: integer
+          readOnly: true
+          description: This Interface's unique ID.
+          example: 123
+          x-linode-cli-display: 1
+        type:
+          type: string
+          description: |
+            The Network Interface type to create. A Network Interface of type:
+
+            * `default` creates a public Network Interface that accepts external Network traffic. If your Linode
+            does not have a public Network Interface it will not receive a public IP address.
```

Comment:
(Unless a boot config lists *no* interfaces. In this case for compatibility the Linode will be provided with a public interface.)